test-clients/src/main/resource/contract/bytecodes/LargeContract.bin

```diff
@@ -0,0 +1 @@
57611fd8612088565b5b50919050565b611fe882612150565b810181811067ffffffffffffffff82111715612007576120066120e6565b5b80604052505050565b600061201b82611f61565b91507fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff82141561204e5761204d612059565b5b600182019050919050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052601160045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052602260045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052603260045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052604160045260246000fd5b600060033d11156121345760046000803e612131600051612161565b90505b90565b600080fd5b600080fd5b600080fd5b600080fd5b600080fd5b6000601f19601f8301169050919050565b60008160e01c9050919050565b7f455243313135353a207472616e7366657220746f206e6f6e204552433131353560008201527f526563656976657220696d706c656d656e746572000000000000000000000000602082015250565b7f455243313135353a204552433131353552656365697665722072656a6563746560008201527f6420746f6b656e73000000000000000000000000000000000000000000000000602082015250565b7f455243313135353a2062616c616e636520717565727920666f7220746865207a60008201527f65726f2061646472657373000000000000000000000000000000000000000000602082015250565b7f455243313135353a2063616c6c6572206973206e6f74206f776e6572206e6f7260008201527f20617070726f7665640000000000000000000000000000000000000000000000602082015250565b7f455243313135353a207472616e7366657220746f20746865207a65726f20616460008201527f6472657373000000000000000000000000000000000000000000000000000000602082015250565b7f455243313135353a207472616e736665722063616c6c6572206973206e6f742060008201527f6f776e6572206e6f7220617070726f7665640000000000000000000000000000602082015250565b7f455243313135353a20696e73756666696369656e742062616c616e636520666f60008201527f72207472616e7366657200000000000000000000000000000000000000000000602082015250565b7f455243313135353a2073657474696e6720617070726f76616c2073746174757360008201527f20666f722073656c660000000000000000000000000000000000000000000000602082015250565b7f455243313135353a206163636f756e747320616e6420696473206c656e67746860008201527f206d69736d617463680000000000000000000000000000000000000000000000602082015250565b7f455243313135353a2069647320616e6420616d6f756e7473206c656e6774682060008201527f6d69736d61746368000000000000000000000000000000000000000000000000602082015250565b600060443d101561249457612517565b61249c611d9d565b60043d036004823e80513d602482011167ffffffffffffffff821117156124c4575050612517565b808201805167ffffffffffffffff8111156124e25750505050612517565b80602083010160043d0385018111156124ff575050505050612517565b61250e82602001850186611fdf565b82955050505050505b90565b61252381611ef7565b811461252e57600080fd5b50565b61253a81611f09565b811461254557600080fd5b50565b61255181611f15565b811461255c57600080fd5b50565b61256881611f61565b811461257357600080fd5b5056fea2646970667358221220af581ac064a50b92c5abaf75f02c2d92d0547d6e0432e5f94b1af42bed54870a64736f6c63430008070033
```

Comment:
Where is the solidity source for this?

Reply:
It's in `CryptoKitties.sol` . Will rename this accordingly.